modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,182 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// link:https://github.com/tektoncd/pipeline/pull/1333(#1333)
+// @savitaashture
+
+[id="cli-new-features-1-8_{context}"]
+=== CLI
+
+* With this update, you can use the `tkn taskrun export` command to export a live task run from a cluster to a YAML file, which you can use to import the task run to another cluster.
+// link:https://github.com/tektoncd/cli/pull/1531[(#1531)]
+// Chmouel Boudjnah @chmouel
+
+* With this update, you can add the `-o name` flag to the `tkn pipeline start` command to print the name of the pipeline run right after it starts.
+// link:https://github.com/tektoncd/cli/pull/1540[(#1540)]
+// Chmouel Boudjnah @chmouel
+
+* This updates adds the list of available plugins to the `tkn --help` command.
+// link:https://github.com/tektoncd/cli/pull/1535[(#1535)]
+// Chmouel Boudjnah @chmouel
+
+* With this update, while deleting a pipeline run or task run, you can use both the `--keep` and `--keep-since` flags together.
+// link:https://github.com/tektoncd/cli/pull/1533[(#1533)]
+// Divyanshu Agrawal @divyansh42
+
+* With this update, you can use the `spec.status` value, `Cancelled`, instead of the deprecated `PipelineRunCancelled` status.
+// link:https://github.com/tektoncd/cli/pull/1554[(#1554)]
+// Vinamra Jain @vinamra28
+
+[id="operator-new-features-1-8_{context}"]
+=== Operator
+
+// [Hub]
+* With this update, as an administrator, you can configure your local Tekton Hub instance to use a custom database rather than the default database.
+// (link:https://issues.redhat.com/browse/SRVKP-2309[SRVKP-2309 Pipeline as code: validate repository URL uniqueness])
+// Puneet Punamiya
+
+* With this update, as a cluster administrator, if you enable your local Tekton Hub instance, it periodically refreshes the database so that changes in the catalog appear in the Tekton Hub web console. You can adjust the period between refreshes.
++
+Previously, to add the tasks and pipelines in the catalog to the database, you performed that task manually or set up a cron job to do it for you.
+// (link:https://issues.redhat.com/browse/SRVKP-2252[SRVKP-2252 Refresh catalogs on schedule])
+// Puneet Punamiya
+
+* With this update, you can install and run a Tekton Hub instance with minimal configuration. This way, you can start working with your teams to decide which additional customizations they might want.
+// (link:https://issues.redhat.com/browse/SRVKP-2250[SRVKP-2250 Install Hub with default configs])
+// Shiv Verma
+
+// [Chains]
+* Previously, the `chains-config` config map for {tekton-chains} in the `openshift-pipelines` namespace was automatically reset to default after upgrading the {pipelines-title} Operator. The current update fixes this issue. Now, you can configure key-value pairs in the `chains-config` config map, and those values persist after upgrading the {pipelines-title} Operator.
+// https://issues.redhat.com/browse/SRVKP-2349
+// Piyush Garg @piyush-garg
```

Comment:
@piyush-garg Please review and approve.

Reply:
https://github.com/openshift/openshift-docs/pull/49137/files#r947640359